oshmem/proc/proc.h

```diff
@@ -71,7 +58,7 @@ struct oshmem_group_t {
     int                         my_pe;
     int                         proc_count;     /**< number of processes in group */
     int                         is_member;   /* true if my_pe is part of the group, participate in collectives */
-    struct oshmem_proc_t      **proc_array;     /**< list of pointers to ompi_proc_t structures
+    struct ompi_proc_t      **proc_array;     /**< list of pointers to ompi_proc_t structures
                                  for each process in the group */
```

Comment:
I mean coding style issue (could you align **proc_array** field with others). Sorry for confusion.